src/reference/sectionB_elements.inc

```diff
@@ -360,21 +362,21 @@ A :code:`math` element information item (referred to in this specification as a
 
 .. container:: issue-math-child
 
-   2. Each element child of a :code:`math` element MUST have an element-type name that is listed in :numref:`Table {number}: {name}<table_supported_mathml_elements>`.
+   2. Each element child of a :code:`math` element MUST have an element-type name that is listed in :numref:`Table {number} {name}<table_supported_mathml_elements>`.
```

Comment:
Tables in the PDF _are_ rendered as `{number}: {name}`

Reply:
... so we put colons everywhere so it's all consistent??

Reply:
No we should be trying to match the heading format

Reply:
But the tables don't have a heading, they have a title that's rendered as `{number}: {name}`

Reply:
So links to tables should have the : ?

Reply:
Yes I meant to say target format.